DEV_README.rst

```diff
@@ -45,7 +45,7 @@ Cut a release candidate
   - tag release
   - push a tag to GitHub
 
-Travis bot runs authomated tests and publish new version on PyPI when
+Travis bot runs automated tests and publish new version on PyPI when
```

Comment:
Right, it's publish the new version when we pushed a new tag but after it passed automated tests. If you'll read above it's says `- push a tag to GitHub` and then part about tests and publishing. So it's a right sentence in this context.